.github/workflows/tests.yml

```diff
@@ -41,7 +43,7 @@ jobs:
 
     - name: Install test dependencies
       run: |
-        sudo apt-get install libgirepository1.0-dev nginx-full
+        sudo apt-get install libgirepository1.0-dev python3-virtualenv nginx-full
```

Comment:
Seems unrelated and probably wrong?

Reply:
Good catch. This seems to be a remnant from the rebase and has been removed in af993c71985ca3ff2aeaa32e5d2cf97a8f4f22b8 already.